plugins/cue_points/quiz/lib/api/filters/KalturaQuizUserEntryNoAnonymousUsersFilter.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @package api
+ * @subpackage filters
+ */
+class KalturaQuizUserEntryNoAnonymousUsersFilter extends KalturaQuizUserEntryFilter
+{
+	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
```

Comment:
I don’t think it makes sense to create class extension for it, it just another filter attribute on KalturaQuizUserEntryFilter, you can call it isAnonymous and if it’s true or false apply userIdNotIn or the userIdIn accordingly.